### PR TITLE
Added Documentation for React-bootstrap

### PR DESCRIPTION
**Pull request by dohsimpson**
If you’re adding a new scraper, please ensure that you have:

- [x] Tested the scraper on a local copy of DevDocs
- [x] Ensured that the docs are styled similarly to other docs on DevDocs

- [x] Added these files to the <code>public/icons/*your_scraper_name*/</code> directory:
  - [x] `16.png`: a 16×16 pixel icon for the doc
  - [x] `16@2x.png`: a 32×32 pixel icon for the doc
  - [x] `SOURCE`: A text file containing the URL to the page the image can be found on or the URL of the original image itself

### CODE REVIEW

File: lib/docs/filters/react_bootstrap/clean_html.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  class ReactBootstrap
+    class CleanHtmlFilter < Filter
+      def call
+        css('.flex-column.d-flex').remove
+        css('header').remove
+        css('.bs-example').remove
+        css('.position-relative').each do |node|
+          code = node.at_css('textarea')
+          code.name = 'pre'
+          code['style'] = code['style'] + '; border: solid 1px;'
```
Comment (simon04) [inline lib/docs/filters/react_bootstrap/clean_html.rb:11]: - [ ] The `.position-relative` does not seem very specific for a code fragment. What about using `css('pre').ancestors('.position-relative').first`?
- [ ] Please extract the raw JSX code from the code fragment and rely on DevDoc's Prism.js syntax highlighting. See other scrapers for some examples. Most of them follow a patter such as: 

```ruby
node.content = node.content
node['data-language'] = 'jsx'
```

File: lib/docs/scrapers/react_bootstrap.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    options[:skip] = %w(
+      react-overlays/
+    )
+
+    options[:replace_paths] = {
+    }
+
+    options[:trailing_slash] = true
+
+    options[:attribution] = <<-HTML
```
Comment (simon04) [inline lib/docs/scrapers/react_bootstrap.rb:24]: Please also add the license to `assets/javascripts/templates/pages/about_tmpl.coffee`

**Review by simon04 — CHANGES_REQUESTED** [review]
Thank you. Please see my review comments.

File: lib/docs/filters/react_bootstrap/clean_html.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+module Docs
+  class ReactBootstrap
+    class CleanHtmlFilter < Filter
```
[the remark this answers is not in the document]
Comment (dohsimpson) [reply lib/docs/filters/react_bootstrap/clean_html.rb:3]: Hi @simon04 , thanks for the review, I have fixed the second bullet point, which is to remove most of the unnecessary classes. But I would like to keep the styling as "simple" please. As the default "simple" stylesheet looks pretty good in my opinion.

**Review by simon04 — APPROVED** [review]
Thank you!

**Pull request merged**